frontend/www/js/omegaup/typeahead.ts

```diff
@@ -100,6 +100,43 @@ function typeahead<T extends { label: string; value: string }>(
     .trigger('change');
 }
 
+export function nominationTypeahead(
+  elem: JQuery<HTMLElement>,
+  cb?: CallbackType<types.NominationListItem>,
+) {
+  if (!cb) {
+    cb = (event: Event, val: types.NominationListItem) =>
+      $(<EventTarget>event.target).val(val.problem.alias);
+  }
+  elem
+    .typeahead<types.NominationListItem>(
+      {
+        minLength: 3,
+        highlight: false,
+      },
+      {
+        source: typeaheadWrapper(
+          (options: { query: string }) =>
+            new Promise<types.NominationListItem[]>((resolve, reject) =>
+              api.QualityNomination.list({ rowcount: 1000, status: 'all' })
+                .then(data => resolve(data.nominations ?? []))
```

Comment:
Esta API ya regresa un array vacío en caso de no encontrar ningún resultado que coincida con la búsqueda, así que puede quedar así:
```suggestion
                .then(data => resolve(data.nominations))
```

Reply:
@pabo99  me marca el mismo error a que si solo pongo las nominaciones

Reply:
Este es el error
ERROR in /opt/omegaup/frontend/www/js/omegaup/typeahead.ts
ERROR in /opt/omegaup/frontend/www/js/omegaup/typeahead.ts(122,39):
122:39 Argument of type 'NominationListItem[] | null' is not assignable to parameter of type 'NominationListItem[] | PromiseLike<NominationListItem[]> | undefined'.
  Type 'null' is not assignable to type 'NominationListItem[] | PromiseLike<NominationListItem[]> | undefined'.
    120 |             new Promise<types.NominationListItem[]>((resolve, reject) =>
    121 |               api.QualityNomination.list({ rowcount: 1000, status: 'all' })
  > 122 |                 .then(data => resolve(data.nominations))
        |                                       ^
    123 |                 .catch(reject),
    124 |             ),
    125 |         ),